### PR TITLE
feat:login-dashboard-event-cms

### PR DESCRIPTION
**Pull request by MNAlfiansyah**
**Related**
https://chat.digitalservice.id/boards/team/nii5pbyw1idwxrxjd8bp56mtkw/bte55h31wfjgp9xgry174ur3mae/vbyyag4aadi8ejgca6gtqzs55gc/c93nyty1ro7g1jfhhoq6i34tbbw

**Changes**
- Login Cms 
image

**Evidence**
title: login dashboard event cms
project: Event Management System
participants: @rayfajars @MNAlfiansyah @johndy2742 @ristianaditya

### CODE REVIEW

File: dashboard/package.json [hunk 1]
```diff
@@ -27,6 +28,8 @@
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-vue": "^9.17.0",
     "husky": "^8.0.0",
+    "next": "^13.5.5",
+    "next-auth": "4.21.1",
```
Comment (doohanas) [inline dashboard/package.json:32]: ### Ask:

Hello @MNAlfiansyah, what is this package for?

File: dashboard/components/Login/Header.vue [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      Silakan login ke akun admin Anda.
+    </p>
+  </div>
+</template>
+
+<script>
+  // import Logo from '~/assets/logo/sapawarga.svg?inline'
+  export default {
+    name: 'HeaderLogin',
+    // components: { Logo },
```
Comment (doohanas) [inline dashboard/components/Login/Header.vue:37]: ### Ask

Hello @MNAlfiansyah, if this comment still using? If you not using this anymore, you can remove this code.

File: dashboard/package.json (hunk 1 above)
Comment (Ibwedagama) [reply dashboard/package.json:32]: hmm, this is a package for Next Js right? I think we should remove them @MNAlfiansyah 🤔

File: dashboard/pages/index.vue [hunk 3]
```diff
@@ -5,7 +5,11 @@
 </template>
 
 <script setup lang="ts">
+  // const { status, data, lastRefreshedAt, getCsrfToken } = useAuth()
+
   definePageMeta({
     title: 'Beranda',
+    middleware: ['auth'],
   })
+  // console.log(status, data, lastRefreshedAt, await getCsrfToken())
```
Comment (Ibwedagama) [inline dashboard/pages/index.vue:14]: hello @MNAlfiansyah please remove unused comment or code

File: dashboard/pages/login/index.vue [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+              >
+                <div
+                  class="ml-4 mr-4 flex items-center justify-between font-lato text-[14px] font-bold"
+                >
+                  Login Akun
+                  <NuxtIcon
+                    name="common/chevron-right"
+                    :class="{
+                      'eas text-white-700 pointer-events-none text-xl ease-in': true,
+                    }"
```
Comment (marsellavaleria19) [inline dashboard/pages/login/index.vue:22]: Hello @MNAlfiansyah  class 'eas' mean 'ease' on tailwind class or what??

Comment (agunghide) [inline dashboard/pages/login/index.vue:22]: Ask:
Where does this `eas` class come from?

File: dashboard/package.json (hunk 1 above)
Comment (doohanas) [reply dashboard/package.json:32]: Yes, i think so. So, We don't need them

Comment (MNAlfiansyah) [reply dashboard/package.json:32]: from the documentation https://sidebase.io/nuxt-auth/getting-started/installation they needed next-auth for using the provider such as keycloak, i will remove the next but i think i'll keep the next-auth for the peer-dependency. what do you think kang @Ibwedagama @doohanas ?

File: dashboard/pages/login/index.vue (hunk 4 above)
Comment (MNAlfiansyah) [reply dashboard/pages/login/index.vue:22]: sorry teh, i dont need that anymore, thanks for the correction🙏

File: dashboard/components/Login/Header.vue (hunk 2 above)
Comment (MNAlfiansyah) [reply dashboard/components/Login/Header.vue:37]: okey kang i'll remove the comment thanks for the correction🙏

File: dashboard/pages/login/index.vue [hunk 5]
```diff
@@ -0,0 +1,55 @@
+<template>
+    <div class="min-h-screen bg-[#E5E5E5]">
+      <div class="flex h-screen items-center justify-center">
+        <div
+          class="h-fit w-[470px] rounded-lg border border-[#EAEAEF] bg-white px-[24px] py-[40px] drop-shadow-md"
+        >
+          <div class="container mx-auto">
+            <HeaderLogin />
```
Comment (agunghide) [inline dashboard/pages/login/index.vue:8]: Nuxt 3 has auto-imports feature for several folder, include `component` folder. So you can just call this component by using its name without call import on `components` option.

ref: 
https://nuxt.com/docs/examples/features/auto-imports
https://nuxt.com/docs/guide/directory-structure/components

Comment (MNAlfiansyah) [reply dashboard/pages/login/index.vue]: thanks for the insight kang, already push the newest update🙏

**Review by marsellavaleria19 — APPROVED** [review]

File: dashboard/package.json (hunk 1 above)
Comment (Ibwedagama) [reply dashboard/package.json]: Ah I see, okay I don't know if they need `next-auth` but what about `next`? @MNAlfiansyah

Comment (Ibwedagama) [reply dashboard/package.json]: I supposed we don't need Next.Js right? 🤔

**Review by Ibwedagama — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

**Review by rayfajars — APPROVED** [review]

**Pull request merged**